awscli/customizations/paginate.py

```diff
@@ -39,19 +39,37 @@
 STARTING_TOKEN_HELP = """
 <p>A token to specify where to start paginating.  This is the
 <code>NextToken</code> from a previously truncated response.</p>
+<p>For more information, see <a
```

Comment:
I can make that change. How about "For usage examples, see ...". Does that work?